channeld/channel_wire.csv

```diff
@@ -101,6 +101,9 @@ channel_got_announcement,1017
 channel_got_announcement,,remote_ann_node_sig,secp256k1_ecdsa_signature
 channel_got_announcement,,remote_ann_bitcoin_sig,secp256k1_ecdsa_signature
 
+# Wait for reply, to make sure it's on disk.(We store the announcement only once!)
+channel_got_announcement_reply,1117
```

Comment:
I don't think we need to wait for the master in this case. Either it stored it and we're happy, or it crashed and we're going to die in a few milliseconds anyway :-)

Reply:
Especially because we suddenly need to buffer other things while waiting for this reply which really is just an optimization.

Reply:
Thank you! I'll remove it